frame/staking/CHANGELOG.md

```diff
@@ -0,0 +1,56 @@
+# Changelog
+
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
+and this project adheres
+to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
+The semantic versioning guarantees cover the interface to the substrate runtime
+which
+includes this pallet as a dependency. This module will also add storage
+migrations whenever
+changes require it. Stability with regard to offchain tooling is explicitly
+excluded from
+this guarantee: For example adding a new field to an in-storage data structure
+will require
+changes to frontends to properly display it. However, those changes will still
+be regarded
+as a minor version bump.
+
+[//]: # TODO(ank4n) these versions should be same as storage version
+
+## [5.0.0] - UNRELEASED
```

Comment:
I wonder if we should not add RELEASED/UNRELEASED tags to make it easier to maintain the chagelogs. Also, in the context of FRAME, some people may think "released" is if the version is part of a runtime upgrade in the relay chain, so it may be confusing. All things considered, I think we could just have the version here.

Reply:
I am now using Changelog only for storage migrations. And also removed the `UNRELEASED` tag.